bookserver/crud.py

```diff
@@ -124,6 +190,21 @@ async def fetch_last_answer_table_entry(
         return rcd.validator.from_orm(res.scalars().first())  # type: ignore
 
 
+async def fetch_last_poll_response(sid: str, course_name: str, poll_id: str):
```

Comment:
I assume `poll_id` is the `div_id` of a poll question? That would be helpful to document.

Reply:
Also, `sid` is the same as `username`, except it also works for the unique identifier generated for students that aren't logged in?

Reply:
Document the return type; I think it's a string containing the last poll response to the given question.